ElastosBountyProgram/front-end/src/module/form/TeamCreateForm/style.scss

```diff
@@ -0,0 +1,11 @@
+@import "../../../style/variable";
+@import "../../../style/color";
+
+.c_userEditFormContainer {
+    
+    .create-btn{
+        border-color: #48ae92;
```

Comment:
What color is this? Please dont use random colors from outside the defined palette.

Reply:
Done